library/neo4j

```diff
@@ -29,12 +29,22 @@ GitRepo: https://github.com/neo4j/docker-neo4j-publish.git
 GitCommit: eb6334f976453d0d8530a67b086ff96875a0742f
 Directory: 3.4.0/enterprise
 
-Tags: 3.3.5, 3.3
+Tags: 3.3.6, 3.3
+GitRepo: https://github.com/neo4j/docker-neo4j-publish.git
+GitCommit: 15b853ffebb036d981002b8f6130a62a432328d4
+Directory: 3.3.5/community
```

Comment:
This directory should be 3.3.6, shouldn't it? (and the one below)

Reply:
oh good catch, thanks!